airbyte-integrations/connectors/source-azure-blob-storage/metadata.yaml

```
@@ -12,7 +12,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: fdaaba68-4875-4ed9-8fcd-4ae1e0a25093
-  dockerImageTag: 0.4.4
+  dockerImageTag: 0.5.4
```

```suggestion
  dockerImageTag: 0.5.0
```

Please bump to `0.5.0` version.

---

🤦 ofc